src/App.jsx

```diff
@@ -1,13 +1,33 @@
 import { useState } from "react";
 import Modal from "./components/UI/Modal/Modal";
-import "./App.css";
+import Cards from "./components/cards/Cards";
+import "./styles/App.css";
```

Comment:
Please check the appropriate turns for imports in react apps
Should be like this:
1. React imports
2. Libraries
3. Custom hooks
4. Components
5. Helpers, functions
6. Styles